ui/app/templates/components/generated-item-list.hbs

```diff
@@ -23,11 +23,9 @@
       <nav class="tabs">
         <ul>
           {{#each tabs as |tab|}}
-            <li>
-              <LinkTo @route={{get tab.routeParams 0}} @model={{get tab.routeParams 1}} data-test-auth-section-tab={{true}}>
-                {{tab.label}}
-              </LinkTo>
-            </li>
+            <LinkTo @route={{get tab.routeParams 0}} @model={{get tab.routeParams 1}} data-test-auth-section-tab={{true}}>
+              {{tab.label}}
+            </LinkTo>
           {{/each}}
```

Comment:
To fix the active class issue.

Reply:
I'm wondering if there are accessibility concerns by not having the `li` tag in an unordered list? I was looking at the [aria list role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/list_role) but it mentions to avoid if possible. I had a look at the _tabs.scss_ file and it looks to me like the link should be getting styled when active and wrapped in an `li` so I wonder if there is a different issue or if the rules should be tweaked or expanded?

Reply:
Good thoughts. I'm seeing this issue in other places, e.g. `ul` with no `li`. Because this was a nice-to-have, I'll remove this from this PR and make a ticket to address in another PR.